pallets/pooled-staking/src/lib.rs

```diff
@@ -98,6 +100,8 @@ pub mod pallet {
         fungible::Credit<<T as frame_system::Config>::AccountId, <T as Config>::Currency>;
     pub type Delegator<T> = <T as frame_system::Config>::AccountId;
 
+    pub type SessionIndex = u32;
```

Comment:
No longer needed?

Reply:
Yes removed in https://github.com/moondance-labs/tanssi/pull/881/commits/461c1c672e4b12c8752138d29eb326f7d38fe3e9